setup.py

```diff
@@ -45,13 +45,16 @@
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.0',
         'Framework :: Django :: 3.1',
+        'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
+        'Framework :: Django :: 4.1',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```

Comment:
Python 3.7 isn't tested on CI, but it seems to be just to simplify the build matrix https://github.com/django-tenants/django-tenants/pull/804